src/utils/SetupUtil.js

```diff
@@ -56,34 +66,48 @@ export default {
     return _retryPromise.promise;
   },
 
-  setup() {
-    return util.isLinux() ? this.nativeSetup() : this.nonNativeSetup();
-  },
-
-  async nativeSetup () {
+  async setup () {
     while (true) {
       try {
-        docker.setup('localhost', machine.name());
-        docker.isDockerRunning();
-
-        break;
+        if (util.isNative()) {
+          localStorage.setItem('setting.useNative', true);
```

Comment:
This prevents the 'native' check on start-up since that's the default. 
If your socket is present (but broken) saving this preference allows the bypass of the socket check